### ISSUE TITLE
[Team] Sub-teams?

### ISSUE THREAD

**Issue by addyosmani** [yeoman/yeoman#1072]
**tl;dr**: would you guys like more structure in the team (e.g https://jquery.org/team/) or are you happy with the flat _do whatevs is interesting_ approach we've got at the moment?

Yeoman's grown to a level where we have a few specific areas that are worked on:
- Core (yo, generator)
- Generators (generator-angular, generator-backbone etc)
- Docs & Site
- Grunt tasks
- Performance

I'm wondering if it makes sense to formalize the above sub-teams to give us a sense of focus and clarity on who is working on what.

_Example_
Someone looks at our homepage and wonders who the best person to speak to about the site. That might be Eric Duran, but it's not something we really list anywhere.

Do we think introducing sub-teams with specific focus areas would be useful? It doesn't really have to change how we work but just highlights who would like to be responsible for what.

Feel free to shoot down if it doesn't really help :)
